src/presentations/indexable-author-archive-presentation.php

```diff
@@ -74,6 +74,23 @@ public function generate_twitter_title() {
 		return $this->title;
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	public function generate_meta_description() {
+		if ( $this->model->description ) {
+			return $this->model->description;
+		}
+
+		$title_option  = 'metadesc-author-wpseo';
```

Comment:
I'm a little confused by the use of `title` here. Also, using both `$title_option` and `$option_title` makes the code quite hard to process.

Reply:
@IreneStr Variables are renamed and documentation is updated.